apps/backend/.env-example

```diff
@@ -26,6 +26,15 @@ OKTA_DOMAIN="<yourdomain>.okta.com"
 OKTA_CLIENTID=<Your Client ID>
 OKTA_CLIENTSECRET=<Your Client Secret>
 
+# Custom OIDC Service
+OIDC_NAME=<What you want your authentication method to be named on the frontend>
+OIDC_ISSUER=<Your OIDC Base URL (including the following /), for example: https://sample.us.auth0.com/>
```

Comment:
Is it actually necessary to require the following slash here?

Reply:
Not actually necessary, will remove